lab2_common.php

```diff
@@ -0,0 +1,31 @@
+
+<?php
+    $StatesArr = array("passive", "passive", "passive", "passive", "passive");
```

Comment:
Не сокращайте лишний раз переменные, IDE всё равно подсказывает названия
```suggestion
    $statesArray = ["passive", "passive", "passive", "passive", "passive"];
```